src/components/modals/export-map-modal.js

```diff
@@ -0,0 +1,96 @@
+// Copyright (c) 2019 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+
+const StyledExportMapModal = styled.div`
+  padding: ${props => props.theme.modalPadding};
+
+  .title {
+    font-weight: 500;
+    color: ${props => props.theme.textColorLT};
+    font-size: 12px;
+  }
+
+  .description {
+    color: ${props => props.theme.textColor};
+    font-size: 11px;
+  }
+`;
+
+const StyledSection = styled.div`
+  margin: 24px 0;
+`;
+
+const StyledInput = styled.input`
+  width: 100%;
+  padding: ${props => props.theme.inputPadding};
+  color: ${props => props.error ? 'red' : props.theme.titleColorLT};
+  height: ${props => props.theme.inputBoxHeight};
+  outline: 0;
+  font-size: ${props => props.theme.inputFontSize};
+  
+  :active,
+  :focus,
+  &.focus,
+  &.active {
+    outline: 0;
+  }
+`;
+
+const propTypes = {
+  exportHtml: PropTypes.object,
+  onExportMapboxAccessToken: PropTypes.func.isRequired
+};
+
+const ExportMapModal = ({
+  exportHtml,
+  onExportMapboxAccessToken
+}) => (
+  <StyledExportMapModal>
+    <div className="export-map-modal">
+      <div className="title">
+        Export your map into a single html file
+      </div>
+      <div className="subtitle">
+        You will be able to save your configuration and data into a single file and download it on your device.
+      </div>
+
+      <StyledSection>
+        <div>
+          Please provide your Mapbox access token
```

Comment:
Maybe add a link to https://docs.mapbox.com/help/how-mapbox-works/access-tokens/

Reply:
<img width="778" alt="screen shot 2019-02-22 at 2 46 55 pm" src="https://user-images.githubusercontent.com/2554552/53275652-bfb58880-36b0-11e9-913c-13939686ecde.png">